### ISSUE TITLE
Несколько веток комментариев на странице

### ISSUE THREAD

**Issue by Skippidar** [modx-pro/Tickets#143]
Делаю возможность общения на одной странице клиента с менеджером и менеджера с автором проекта (2 разные вкладки на одной странице). Задал 'thread' => 'resource-'~$_pls['id']~'-customer' первому и 'thread' => 'resource-'~$_pls['id']~'-author' второму. Ветки комментариев при загрузке страницы грузятся как надо, но при работе с комментариями (добавлении например) - AJАX'ом обновляется только первая вкладка. Посмотрел JS - там обращение идет напрямую по id самих элементов, id ветки комментариев не используется, как я понимаю.

Собственно вопрос - я правильно понимаю, что такой принцип работы не предусмотрен и нужно самому реализовать обновление формы согласно указанному thread или я просто что-то упустил?

**Comment by Ibochkarev** [comment]
Да, в 2016 году JS действительно ходил по голым `#comment-form` / `#comments`, поэтому на одной странице работала только первая ветка.

Сейчас это починено (с 1.14.0):
- обёртка `.comments-thread` с уникальным id;
- AJAX (save / preview / reply / getlist / tpanel) идёт в свою ветку через `Tickets.threadId`;
- у tpanel классы вместо дублирующихся id.

См. https://github.com/modx-pro/Tickets/pull/193 и follow-up в https://github.com/modx-pro/Tickets/pull/211.

Закрываю: запрос закрыт в релизе 1.14.0.
